Spigot-Server-Patches/0397-Call-PlayerRaidTriggerEvent-before-raid-is-added-to-.patch

```diff
@@ -0,0 +1,28 @@
+From 0a0589ef0a8dbd83497ceecc184a4410d759ed8d Mon Sep 17 00:00:00 2001
+From: simpleauthority <jacob@algorithmjunkie.com>
+Date: Tue, 28 May 2019 02:30:16 -0700
+Subject: [PATCH] Call PlayerRaidTriggerEvent before raid is added to tracker;
```

Comment:
Rename the patch title to smth like "Implement PlayerRaidTriggerEvent" or "Call PlayerRaidTriggerEvent", the current name is way too long. you can still add the long explanation to the commit message.

Reply:
Renamed to `Implement PlayerTriggerRaidEvent`